components/brave_wallet_ui/page/screens/send/send-page/send-screen.tsx

```diff
@@ -39,6 +39,7 @@ export const SendScreen = () => {
         setSelectedSendOption={setSelectedSendOption}
         showSelectTokenModal={showSelectTokenModal}
         selectTokenModalRef={selectTokenModalRef}
+        isAndroid={true}
```

Comment:
Lets also add the optional `isAndroid` prop to the `SendScreen` component and then pass it down to the `Send` component.

```
interface Props {
  isAndroid?: boolean
}

export const SendScreen = ({ isAndroid }: Props) => {
```


```suggestion
        isAndroid={isAndroid}
```